src/artifacts/mod.rs

```diff
@@ -54,7 +54,7 @@ const SOLIDITY: &str = "Solidity";
 const YUL: &str = "Yul";
 
 /// Input type `solc` expects
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, Default)]
```

Comment:
we should implement default manually and set the language to `SOLIDITY.to_string()`

Reply:
> we should implement default manually and set the language to `SOLIDITY.to_string()`

Thank you Mat, does this looks better?